### ISSUE TITLE
Consider displaying web page content, or video, when user is on menu overlay

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#1057]
### Why/User Benefit/User Problem
via Carter: when users press the menu button to decide to go somewhere else, they may want to continue watching the content they're currently seeing. For example, when you press the guide button on other services, like DirectTV, a PIP window will continue showing the TV channel as you look through the guide.

We partially implemented this idea (see through the navigation overlay) but with the home tiles merging into the overlay, we can no longer use this solution. We should consider alternatives.

**Comment by liuche** [comment]
In the new mocks from @aminalhazwani, the menu slides in from the top and bottom so that web content is still shown below. I'm not sure where these mocks are, but there are mocks and we just need to pick a milestone to build this in!

**Comment by aminalhazwani** [comment]
@liuche yeah, that is correct, the mocks are not avaiable yet as I am still wondering on how to best approch those changes, going to share more soon 🙂
